README.md

```diff
@@ -6,7 +6,12 @@ Running latest all in one image
 docker run -d --name translate-all-in-one \
   -p 8080:8080 \
   -p 16686:16686 \
-  -e TRANSLATE_OTHER_GOOGLE_TRANSLATE_API_KEY={GoogleTranslate API key} \
+  -e TRANSLATE_OTHER_GOOGLE_PROJECT_ID={Google project id} \
+  -e TRANSLATE_OTHER_GOOGLE_LOCATION={Google location} \
+  -e TRANSLATE_OTHER_AWS_ACCESS_KEY={AWS API access key} \
+  -e TRANSLATE_OTHER_AWS_SECRET_KEY={AWS API secret key} \
```

Comment:
Was there `API` in the text before? Env variable names were changed and are not up to date.

```suggestion
  -e TRANSLATE_OTHER_AWS_ACCESS_KEY_ID={AWS access key id} \
  -e TRANSLATE_OTHER_AWS_SECRET_ACCESS_KEY={AWS secret access key} \
```